StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -8,8 +8,54 @@
 import Foundation
 @_spi(STP) import StripeCore
 
-protocol FinancialConnectionsAPIClient {
+class FinancialConnectionsAPIClient {
+    let backingAPIClient: STPAPIClient
 
+    var isLinkWithStripe: Bool = false
+    var consumerPublishableKey: String?
+    var consumerSession: ConsumerSessionData?
+
+    static let shared = FinancialConnectionsAPIClient(apiClient: .shared)
+
+    init(apiClient: STPAPIClient) {
+        self.backingAPIClient = apiClient
+    }
+
+    func consumerPublishableKeyProvider(canUseConsumerKey: Bool) -> String? {
+        guard canUseConsumerKey, isLinkWithStripe, consumerSession?.isVerified == true else {
+            return nil
+        }
+        return consumerPublishableKey
+    }
+
+    func get<T: Decodable>(
+        resource: String,
+        parameters: [String: Any],
+        canUseConsumerPublishableKey: Bool
```

Comment:
feel 100% free to ignore this:

maybe the name would be more idiomatic/clear if it was something like `useConsumerPublishableKeyIfNeeded`

the name I suggest is unfortunately more lengthy, however

I just use "if needed" because there's some Apple API's with it - ex. `layoutIfNeeded()`

`canUseConsumerPublishableKey` does say the same thing, but for some reason feels more confusing (though, again, feel 100% free to ignore my suggestion - not super bullish on my own name either)

Reply:
Will update to `useConsumerPublishableKeyIfNeeded`!